client/src/modules/email-report/modal/categories-people-list-service.js

```diff
@@ -0,0 +1,41 @@
+angular.module('bhima.services')
+    .service('CategoriesPeopleService', CategoriesPeopleService);
+
+CategoriesPeopleService.$inject = [
+    '$http', 'util', 'SessionService', '$uibModal',
+    'DocumentService', 'VisitService', 'DepricatedFilterService',
+];
+
+/**
+ * @module EmailReportService
+ *
+ * This service is responsible for providing an interface between angular
+ * module controllers and the server /tools/bed/ API.
+ */
+function CategoriesPeopleService($http, util, Session, $uibModal,
+    Documents, Visits, Filters) {
+
+    var service = this;
+    service.list = [];
+
+    init();
```

Comment:
There is no need for this `init` method.  We use them in our controllers to group the startup functions together (since a controller is initialized each page navigation in angular).

[AngularJS services are singletons](https://stackoverflow.com/questions/32677694/angular-js-services-are-singleton-objects-means), which means they are only created once.  Since you aren't doing any complex logic, just bind everything to the service directly, like this:
```js
var service = this;

service.list = [{
   table: 'employee',
   // etc ...
}];

return service;
```